### PR TITLE
feat(solace): update solace finance integration

### PR DESCRIPTION
**Pull request by leonardishere**
## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] (optional) As a contributor, my Ethereum address/ENS is:
- [ ] (optional) As a contributor, my Twitter handle is:

### Solace Portfolio Insurance V3

http://localhost:5001/apps/solace/positions?groupIds[]=policies&network=ethereum
http://localhost:5001/apps/solace/balances?addresses[]=0xfb5cAAe76af8D3CE730f3D62c6442744853d43Ef&network=ethereum

http://localhost:5001/apps/solace/positions?groupIds[]=policies&network=aurora
no active holders

http://localhost:5001/apps/solace/positions?groupIds[]=policies&network=polygon
http://localhost:5001/apps/solace/balances?addresses[]=0xF6DA0e773fa4095b8454E155A23446D8eBC068cF&network=polygon

http://localhost:5001/apps/solace/positions?groupIds[]=policies&network=fantom
http://localhost:5001/apps/solace/balances?addresses[]=0xa4C86a9706605611A6954A5F70fE9f8Afc5018ce&network=fantom

### Aurora

http://localhost:5001/apps/solace/positions?groupIds[]=bonds&network=aurora
http://localhost:5001/apps/solace/balances?addresses[]=0x44286e05Ca69D97DB77A281B2B22bE2eB70e6EDC&network=aurora

http://localhost:5001/apps/solace/positions?groupIds[]=xslocker&network=aurora
http://localhost:5001/apps/solace/balances?addresses[]=0xa4C86a9706605611A6954A5F70fE9f8Afc5018ce&network=aurora

### Fantom

http://localhost:5001/apps/solace/positions?groupIds[]=bonds&network=fantom
http://localhost:5001/apps/solace/balances?addresses[]=0xa4C86a9706605611A6954A5F70fE9f8Afc5018ce&network=fantom

http://localhost:5001/apps/solace/positions?groupIds[]=xslocker&network=fantom
http://localhost:5001/apps/solace/balances?addresses[]=0xf9af2e7927577d366291d8986dBD15daCEf676a6&network=fantom

### CODE REVIEW

**Comment by wpoulin** [comment]
Hey hi,

I just want to double check you and your team are not confused with the recent code changes we applied to your app integration ? 

Plus, I think the policies and xsLocker positions are counting towards TVL right ? if so, do you mind adding liquidity on the `dataProps` and `displayProps`. Also, don't forget to enable it to count towards TVL with the `options: { includeInTvl: true }` on the decorator.

**Comment by leonardishere** [comment]
> I just want to double check you and your team are not confused with the recent code changes we applied to your app integration ?
Are you talking about this one? https://github.com/Zapper-fi/studio/pull/692
I duplicated your logic for the new chains. Am I missing something?

> Plus, I think the policies and xsLocker positions are counting towards TVL right ? if so, do you mind adding liquidity on the dataProps and displayProps. Also, don't forget to enable it to count towards TVL with the options: { includeInTvl: true } on the decorator.
I didn't see that was an option, I just saw the TVL fetcher disappear and didn't know how it was still returning data. Will add

**Comment by wpoulin** [comment]
yes I'm taking about this PR. No, you did great ! I just wanted to make sure you saw the changes and were not lost/confused

> Edit: it looks like you added some stuff to Ethereum and not Polygon, and I duplicated the Polygon logic to new chains. fixing

Yup, I totally forgot to add the logic on the Polygon implementation 😬 

> I didn't see that was an option, I just saw the TVL fetcher disappear and didn't know how it was still returning data. Will add

I know Tvl-Fetcher is kinda new, but things are moving quickly and we will be deprecating it pretty soon

**Review by wpoulin — APPROVED** [review]

**Comment by leonardishere** [comment]
>  I just wanted to make sure you saw the changes and were not lost/confused

there was some initial confusion but nothing that couldn't be figured out. thanks for cleaning up our integration btw, I saw you deduplicated the helpers amongst other improvements

File: src/apps/solace/helpers/SolacePolicyBalanceHelper.ts [hunk 1]
```diff
@@ -8,6 +8,9 @@ import { Network } from '~types/network.interface';
 import { SolaceContractFactory } from '../contracts';
 import { SOLACE_DEFINITION } from '../solace.definition';
 
+const SOLACE_COVER_POINTS_ADDRESS = '0x501ace72166956f57b44dbbcc531a8e741449997';
+const SOLACE_COVER_PRODUCT_ADDRESS = '0x501ACeB72d62C9875825b71d9f78a27780B5624d';
```
Comment (wpoulin) [inline src/apps/solace/helpers/SolacePolicyBalanceHelper.ts:12]: Can you lower case this address please

**Comment by wpoulin** [comment]
Good job ! Merge main and it's a go, i'll merge it right away

**Pull request merged**